ansible_events_ui/db/models/activation.py

```diff
@@ -52,8 +52,7 @@
         sa.ForeignKey("playbook.id", ondelete="CASCADE"),
         nullable=False,
     ),
-    sa.Column("activation_status", sa.String),
-    sa.Column("activation_enabled", sa.Boolean, nullable=False),
+    sa.Column("activation_status", sa.Boolean, nullable=False),
```

Comment:
Why the status is boolean?

Reply:
I first thought `activation_status` and `activation_enabled` are two different fields, but looking at this ([AAP-5227](https://issues.redhat.com/browse/AAP-5227)), it seems to me that they are one field, set to either enabled or disabled. Maybe, I'm wrong; could you tell me more on how this works?